src/frontend/components/data-table/lib/component.js

```diff
@@ -290,21 +290,21 @@ class DataTableComponent extends Component {
     const $searchInput = $(`<input class='form-control form-control-sm' type='text' placeholder='Search' value='${searchValue}'/>`)
     $searchInput.appendTo($('.input', $searchElement))
     if (col.typeahead_use_id && searchValue) {
+      console.log("searchInput: ", $searchInput)
       $searchInput.after(`<input type="hidden" class="search">`)
-      $.ajax({
-        type: 'GET',
-        url: this.getApiEndpoint(columnId) + searchValue + '&use_id=1',
-        dataType: 'json'
-      }).done(function(data) {
-        if (!data.error) {
+      const response = await fetch(this.getApiEndpoint(columnId) + searchValue + '&use_id=1')
```

Comment:
The AJAX request caused the table to refresh twice - I have opted to use Fetch as this allows us to await the response before rendering (`$.ajax` allows the function to return whilst still processing the request)